atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/api/cache/CacheStoreImplTest.java

```diff
@@ -49,7 +49,7 @@ public void updatesToSnapshotStoreReflectedInCacheStore() {
         snapshotStore.storeSnapshot(
                 Sequence.of(5L),
                 ImmutableSet.of(TIMESTAMP_2),
-                ValueCacheSnapshotImpl.of(HashMap.empty(), HashSet.empty(), ImmutableSet.of()));
+                ValueCacheSnapshotImpl.of(HashMap.empty(), HashSet.empty(), HashSet.empty(), ImmutableSet.of()));
```

Comment:
Many tests have empty snapshots - in a separate PR, I might pull out some constant to represent this.